src/base/TilesRendererBase.js

```diff
@@ -308,17 +300,39 @@ export class TilesRendererBase {
 				delete tile.content.boundingVolume;
 
 			}
+			uri = tile.content?.uri;
 
 		}
 
 		tile.parent = parentTile;
 		tile.children = tile.children || [];
 
-		const uri = tile.content && tile.content.uri;
+		if ( tile.implicitTiling ) {	//only for root
+
+			//	Store the infos from the tileset
+			tile.availableLevels = tile.implicitTiling.availableLevels;
+			tile.subdivisionScheme = tile.implicitTiling.subdivisionScheme;
+			tile.__subtreeLevels = tile.implicitTiling.subtreeLevels;
+			tile.subtrees = tile.implicitTiling.subtrees;
+			tile.__subtreeDivider = tile.subdivisionScheme === "QUADTREE" ? 4 : 8;
+			tile.__subtreeUri = tile.implicitTiling.subtrees.uri;
+			tile.__contentUri = tile.content?.uri;
+
+			//	Declare some properties
```

Comment:
nit - can we use spaces rather than tabs after `//` like the rest of the comments